external/wpgraphql.php

```diff
@@ -312,14 +327,25 @@ function updateGraphQlFields() {
 
 function cptui_graphql_init() {
 	if ( class_exists( 'WPGraphQL_CPT_UI' ) ) {
-		add_action( 'admin_notices', function () {
-			$link = trailingslashit( admin_url() ) . 'plugins.php';
-			?>
+		add_action(
+			'admin_notices',
+			function () {
+				$link = trailingslashit( admin_url() ) . 'plugins.php';
+				?>
 			<div class="notice notice-error">
-				<p><?php echo sprintf( __( 'Custom Post Type UI has native support for WPGraphQL. Please <a href="%s">de-active</a> the "WPGraphQL for Custom Post Type UI" extension to proceed.', 'custom-post-type-ui' ), $link ); ?></p>
+				<p>
+				<?php
+				echo sprintf(
+					// phpcs:ignore.
+					esc_attr__( 'Custom Post Type UI has native support for WPGraphQL. Please <a href="%s">de-active</a> the "WPGraphQL for Custom Post Type UI" extension to proceed.', 'custom-post-type-ui' ),
```

Comment:
This should be `esc_html__` instead as it's going inside a `<p>` tag and not as an actual attribute.